styles.css

```diff
@@ -133,3 +135,937 @@ li {
   left: 223.1px;
   top: 317px;
 }
+
+/* Header section style end start here */
+
+/* ------------------------------------------------------------------------------------------------------------------ */
+
+/* Work section style start here */
+
+.work-section {
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  width: 375px;
+  height: 3455px;
+  padding: 120px 0;
+  left: 0;
+  top: 682px;
+  align-items: center;
+  background-color: #fff;
+}
+
+.title {
+  position: static;
+  width: 327px;
+  height: 83px;
+  left: 0;
+  top: 0;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  color: #1c1a19;
+}
+
+.vector-line {
+  position: static;
+  width: 327px;
+  height: 0;
+  left: 0;
+  top: 107px;
+  border: 1px solid #212121;
+  margin-bottom: 80px;
+}
+
+.Img-Placeholder {
+  position: static;
+  width: 327px;
+  height: 237px;
+  left: 24px;
+  top: 0;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 0 0;
+}
+
+.title-post {
+  position: static;
+  width: 327px;
+  height: 0;
+  left: 0;
+  top: 0;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  color: #091e42;
+}
+
+.supporting-text {
+  position: static;
+  width: 327px;
+  height: 147px;
+  left: 0;
+  top: 48px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin: 12px 0;
+}
+
+.tags {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  position: static;
+  width: 297px;
+  height: 40px;
+  left: 0;
+  top: 207px;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+  margin: 12px -17px;
+}
+
+.tag-1 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  position: static;
+  width: 51px;
+  height: 40px;
+  left: 0;
+  top: 0;
+  background: #fff;
+  border: 1px solid #8993a4;
+  box-sizing: border-box;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-right: -8px;
+}
```

Comment:
Text of the technologies of the project  should be center as the Figma design, I suggest doing that

- [ ] Your design 
![image](https://user-images.githubusercontent.com/61790587/164911891-3305fc64-5fe3-4653-b699-9da64f493885.png)

- [ ] Figma design 
![image](https://user-images.githubusercontent.com/61790587/164911908-82dad970-0b45-4619-a869-4ea792296fd8.png)